plugins/qlcplus.js

```diff
@@ -10,7 +10,7 @@ module.exports.export = function exportQLCplus(library, options) {
   const defaults = require(path.join(options.baseDir, 'fixtures', 'defaults'));
 
   for (const data of library) {
-    const fixture = Object.assign({}, defaults, JSON.parse(fs.readFileSync(path.join(options.baseDir, 'fixtures', data.manufacturerKey, data.fixtureKey + '.json'), 'utf-8')));
+    let fixture = Object.assign({}, defaults, JSON.parse(fs.readFileSync(path.join(options.baseDir, 'fixtures', data.manufacturerKey, data.fixtureKey + '.json'), 'utf-8')));
```

Comment:
Why `let`? I don't see any additional occurrence where the value of `fixture` is set.